package.json

```diff
@@ -51,8 +51,12 @@
     "typescript": "^4.4.4"
   },
   "dependencies": {
+    "csv-string": "^4.0.1",
+    "date-fns": "^2.25.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "prettier": "^2.4.1"
+    "js-file-download": "^0.4.12",
```

Comment:
Is js-file-download still usefull in core lib ?

Reply:
Yes, I just added it to provide solutions with the function `downloadFileFromData`